src/esp/assets/ResourceManager.h

```diff
@@ -374,9 +344,35 @@ class ResourceManager {
     return collisionMeshGroups_.at(collisionAssetHandle);
   }
 
+  // /**
+  //  * @brief Return manager for construction and access to asset attributes.
+  //  */
+  // managers::AssetAttributesManager::ptr getAssetAttributesManager() {
+  //   return assetAttributesManager_;
+  // }
+  // /**
+  //  * @brief Return manager for construction and access to object attributes.
+  //  */
+  // managers::ObjectAttributesManager::ptr getObjectAttributesManager() {
+  //   return objectAttributesManager_;
+  // }
+  /**
+   * @brief Return manager for construction and access to physics world
+   * attributes.
+   */
+  managers::PhysicsAttributesManager::ptr getPhysicsAttributesManager() {
+    return physicsAttributesManager_;
+  }
+  /**
+   * @brief Return manager for construction and access to scene attributes.
+   */
+  managers::SceneAttributesManager::ptr getsceneAttributesManager() {
```

Comment:
```suggestion
  managers::SceneAttributesManager::ptr getSceneAttributesManager() {
```

Reply:
Oh for crying out loud.  my camel case lost a hump :D